### PR TITLE
Typos and wording update v2

### PR DESCRIPTION
**Pull request by lesteryee**
2nd round of updates based upon the main branch

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [portal.envoyai.com](https://portal.envoyai.com)
 
 #### How do I learn how to create a machine?
-The best way to learn about how to create a machine on the EnvoyAI Platform is to read the [helloworld walk-through](./test-helloworld/README.md).
+The best way to learn about how to create a machine on the EnvoyAI Platform is to explore the [helloworld walk-through](./test-helloworld/README.md).
```
Comment (jaketaylorpro) [inline FAQ.md]: unnecessary, lets revert that

File: FAQ.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ To learn more about advanced input and output types, please see the [test-echo](
 We reccomend using Google Drive, or Amazon S3 to publicly host sample images and data. You can then publish a link to that data in your Machine description.
 
 #### How do I quickly adapt an already working dockerfile?
-To see an example that uses a trained network to do image recognition, see the caffe example we adapated for our platform 
+To see an example that uses a trained network to do image recognition, check out the caffe example we adapated for our platform
```
Comment (jaketaylorpro) [inline FAQ.md]: unnecessary, lets revert that

File: README.md [hunk 3]
```diff
@@ -31,18 +31,18 @@ The data that your algorithm evaluates is called its _inputs_, the data your alg
 Both input and outputs must have well defined data type definitions called a _schema_.
 The EnvoyAI Platform is based on [JSON-Schema](http://json-schema.org) but uses different defaults, and has additional 
 features to describe files and DICOM studies. Additionally the platform uses a similar schema to define how the inputs 
-and outputs are to be displayed. Either JSON or YAML formatted schemas are accepted.
+and outputs are to be displayed. Both JSON and YAML formatted schemas are accepted.
```
Comment (jaketaylorpro) [inline README.md]: Either is more accurate here I think

File: test-echo/README.md [hunk 4]
```diff
@@ -40,10 +40,10 @@ image.save('/envoyai/output/image.png','PNG')
 
 ## 2 Input
 
-In practice, all inputs defined by your algorithms `com.envoyai.schema-in` will be supplied via JSON over WebAPI via the 
-'EnvoyAI Integration API', and one should not be too concerned with the exact user interface, as it may need to vary 
-system to system, or client to client. However for testing and demonstration purposes, at https://portal.envoyai.com 
-we dynamically render input controls for any input type.
+In practice, all inputs defined by your algorithm `com.envoyai.schema-in` will be supplied via JSON over HTTP/HTTPS via the
```
Comment (jaketaylorpro) [inline test-echo/README.md]: WebApi is clearer than HTTP/HTTPS

File: test-echo/README.md [hunk 5]
```diff
@@ -40,10 +40,10 @@ image.save('/envoyai/output/image.png','PNG')
 
 ## 2 Input
 
-In practice, all inputs defined by your algorithms `com.envoyai.schema-in` will be supplied via JSON over WebAPI via the 
-'EnvoyAI Integration API', and one should not be too concerned with the exact user interface, as it may need to vary 
-system to system, or client to client. However for testing and demonstration purposes, at https://portal.envoyai.com 
-we dynamically render input controls for any input type.
+In practice, all inputs defined by your algorithm `com.envoyai.schema-in` will be supplied via JSON over HTTP/HTTPS via the 
+'EnvoyAI Integration API'.  One should not be concerned with the exact user interface presentation/experience, as it may need to vary
```
Comment (jaketaylorpro) [inline test-echo/README.md]: I don't like experience here

File: test-helloworld/README.md [hunk 6]
```diff
@@ -95,15 +95,14 @@ fda_status: N/\n"
...
+The EnvoyAI Platform supports Nvidia CUDA 8 and CuDNN 5. If your algorithm require a different version, please contact us and we will try to make it available.
 
-Use the __LABEL__ `com.envoyai.nvidia` to toggle on CUDA support.
+Use the __LABEL__ `com.envoyai.nvidia` to enable CUDA support.
 ```Dockerfile
 LABEL com.envoyai.nvidia=true
 ```
 
-You will likely want to base your Docker image off of an NVidia example.
+You will likely want to base your Docker image off of an Nvidia example.
```
Comment (jaketaylorpro) [inline test-helloworld/README.md]: its actually nVidia, we both got it wrong

**Review by jaketaylorpro — CHANGES_REQUESTED** [review]
hey lester, a lot of good typos fixed. lets revert a few of the unproductive changes before we can merge

File: FAQ.md (hunk 1 above)
Comment (lesteryee) [reply FAQ.md]: OK

File: FAQ.md (hunk 2 above)
Comment (lesteryee) [reply FAQ.md]: OK

File: README.md (hunk 3 above)
Comment (lesteryee) [reply README.md]: OK

File: test-echo/README.md (hunk 4 above)
Comment (lesteryee) [reply test-echo/README.md]: OK

File: test-echo/README.md (hunk 5 above)
Comment (lesteryee) [reply test-echo/README.md]: Fixed as well as the HTTP/HTTPS -> WebApi

File: test-helloworld/README.md (hunk 6 above)
Comment (lesteryee) [reply test-helloworld/README.md]: OK.  I checked their website and it is all uppercase.  Historically it has been as described nVidia.  I believed they have rebranded to uppercase with secondary branding as "nVidia".

**Review by jaketaylorpro — APPROVED** [review]

**Pull request merged**